src/tallies/tally.cpp

```diff
@@ -662,6 +677,18 @@ Tally::score_name(int score_idx) const {
   return reaction_name(scores_[score_idx]);
 }
 
+std::string
+Tally::nuclide_name(int nuclide_idx) const {
+  if (nuclide_idx != -1) {
+    return "total";
```

Comment:
Unless I'm misunderstanding something, I think this check should be on the value of `nuclide` below, not on `nuclide_idx`. The default value of `nuclides_` in a tally is a vector with a single element storing -1.

Reply:
Ah, yeah totally. Good catch. Thanks!